tsdb/head.go

```diff
@@ -1082,7 +1082,7 @@ func (h *Head) SetMinValidTime(minValidTime int64) {
 // Truncate removes old data before mint from the head and WAL.
 func (h *Head) Truncate(mint int64) (err error) {
```

Comment:
Is there a reason why you did not pass the context here as well?

Reply:
Mainly I couldn't see where this was called except from `reload`, where we don't seem to have a useful context.
But I can add it for consistency.

Reply:
Have now added.